scene/gui/rich_text_label.h

```diff
@@ -392,7 +392,8 @@ class RichTextLabel : public Control {
 	void _find_click(ItemFrame *p_frame, const Point2i &p_click, ItemFrame **r_click_frame = nullptr, int *r_click_line = nullptr, Item **r_click_item = nullptr, int *r_click_char = nullptr, bool *r_outside = nullptr);
 
 	String _get_line_text(ItemFrame *p_frame, int p_line, Selection p_sel) const;
-	bool _search_line(ItemFrame *p_frame, int p_line, const String &p_string, Item *p_from, Item *p_to);
+	bool _search_line(ItemFrame *p_frame, int p_line, const String &p_string, int char_idx, bool reverse_search);
```

Comment:
`char_idx` and `reverse_search` need to be preficed with `p_` ditto for `_search_table`

Reply:
I thought the `p_` prefix is used to denote a pointer. Does `p_` mean something else in godot?